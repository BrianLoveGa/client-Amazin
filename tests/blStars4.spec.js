import React from "react";
import GoodStar from "../src/components/RatingSummary/goodStar";
import renderer from "react-test-renderer";

// snapshot test if anything in component is changed this snapshot will need to be updated or this test changed

describe("This snapshot test should pass for 4 stars", () => {
  it("Should render as expected!", () => {
    const tree = renderer.create(<GoodStar />).toJSON();

    console.log(tree);

    expect(tree).toMatchSnapshot();
  });
});