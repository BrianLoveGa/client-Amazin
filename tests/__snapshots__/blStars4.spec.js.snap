// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This snapshot test should pass for 4 stars Should render as expected! 1`] = `
<div
  className="goodstar"
>
  <span
    className="gold"
    style={
      Object {
        "color": "rgb(255, 188, 40)",
      }
    }
  >
    ★
  </span>
  <span
    className="gold"
    style={
      Object {
        "color": "rgb(255, 188, 40)",
      }
    }
  >
    ★
  </span>
  <span
    className="gold"
    style={
      Object {
        "color": "rgb(255, 188, 40)",
      }
    }
  >
    ★
  </span>
  <span
    className="gold"
    style={
      Object {
        "color": "rgb(255, 188, 40)",
      }
    }
  >
    ★
  </span>
  <span
    className="gray"
    style={
      Object {
        "color": "gray",
      }
    }
  >
    ★
  </span>
  <span
    className="tiny"
    style={
      Object {
        "fontSize": "10px",
        "padding": "0px 20px 0px 5px",
        "position": "relative",
        "top": "-8px",
      }
    }
  >
     
    v
     
  </span>
  <a
    href="#"
    style={
      Object {
        "fontSize": "13px",
      }
    }
  >
     26,547 ratings
  </a>
  <span
    className="line"
    style={
      Object {
        "fontSize": "13px",
        "padding": "10px",
      }
    }
  >
     | 
  </span>
  <a
    href="#"
    style={
      Object {
        "fontSize": "13px",
      }
    }
  >
    800 answered questions
  </a>
</div>
`;
